var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -39,6 +41,11 @@ class Subversion(AutotoolsPackage):
     # Installation has race cases.
     parallel = False
 
+    def url_for_version(self, version):
+        """Handle Subversion's new release URL versions"""
```

Comment:
Are the old versions available at this new URL? If so, we can update the `url` above or add a `list_url` if we need to. If not, the `url_for_version` needs to support all versions.

Reply:
If it makes sense we can also turn the single url into [urls](https://spack.readthedocs.io/en/latest/packaging_guide.html#mirrors-of-the-main-url).

Reply:
Old versions aren't available at new URL, so any subversion > 1.12 from new URL otherwise at old URL. `urls` is the solution but it's not available in spack `.0.13.2`, so it's gonna fail in old versions of spack. I put condition in `url_for_version`

Reply:
```
        if version > Version('1.12.2'):
            url = 'https://downloads.apache.org/subversion'
        else:
            url = 'https://archive.apache.org/dist/subversion'
        return url + '/subversion-%s.tar.gz' % (version)
```

Reply:
> Old versions aren't available at new URL, so any subversion > 1.12 from new URL otherwise at old URL. urls is the solution but it's not available in spack .0.13.2, so it's gonna fail in old versions of spack. I put condition in url_for_version

The builtin package repository is bundled with Spack and this will be merged to `develop` (which is after 0.15.1). In my opinion if `urls` seems a solution I think we should go with that.

Reply:
Thanks @ajkotobi